yt/utilities/load_sample.py

```diff
@@ -45,12 +48,21 @@ def load_sample(name=None, specific_file=None):
     base_path = fido.fido.path
     fileext, name, extension = _validate_sampledata_name(name)
 
+    downloader = None
+    if pbar:
+        try:
+            import tqdm  # noqa: F401
+            from pooch import HTTPDownloader
```

Comment:
I tried your PR and it works! However, if pooch isn't importing properly the error message will still default to the tqdm one. Is there a way that this can be made clearer? 


Also maybe you want to use the on demand import of pooch defined at line 12?

Reply:
I don't think this can happen since if you attempt to use `yt.load_sample` while pooch isn't installed, it will raise :
```python
ImportError: This functionality requires the pooch package to be installed.
```
whether or not tqdm is installed. (I just checked this :))

Reply:
And I just switched to the on-demand-import for pooch, thanks for catching this !

Reply:
I made a suggestion to avoid any pooch-related imports, just in case! I think you'll need to use pch.pooch to access the HTTPDownloader.

Reply:
approved, it's clearer this way and I like that it avoids imports outside of top-level (except for tqdm which I don't see a solution to)

Reply:
Yeah I think the tqdm import here is a good solution. 🙂  I can't think of an alternative solution either.